source/options.css

```diff
@@ -46,54 +30,42 @@ hr {
 
 label {
 	display: flex;
-	width: 100%;
-	margin: 0.5em 0 0;
 	align-items: center;
 	flex-wrap: wrap;
-}
-
-summary label {
-	display: inline;
+	width: 100%;
+	margin: 0.5em 0;
 }
 
 label input[type='checkbox'] {
 	margin-right: 0.5em;
 }
 
-/* Workaround for missing indeterminate icon bug in chrome extension */
-input[type='checkbox']:indeterminate::after {
-	content: '\2212';
-	font-size: 16px;
-	top: -5px;
-	font-weight: bold;
-	position: absolute;
-}
-
-input {
-	font-family: inherit;
-}
-
-input[type='url'],
-input[type='text'] {
-	display: block;
-	width: 100%;
-	padding: 0.25em;
+/*
+Monospace all text inputs
+We don't have many text inputs, so this should be fine
+*/
+input:not([type='checkbox']) {
+	padding: 0.25rem;
+	font-family: SFMono-Regular, Consolas, 'Liberation Mono', Menlo, Courier, monospace;
+	font-size: 11px;
 }
 
 input:not([type='checkbox']):not(:placeholder-shown):valid {
 	color: var(--github-green);
-	border: 1px solid var(--github-green);
+	border: 1px solid;
 }
 
 input:not([type='checkbox']):invalid {
 	color: var(--github-red);
 	border: 1px solid;
 }
 
+/*
+Show only a part of the token
+https://github.com/sindresorhus/refined-github/issues/1374#issuecomment-397906701
+*/
 input[type='text'][name='token'] {
-	font-family: Consolas, monospace;
-	font-size: 12px;
-	width: 35ch; /* Show only a part of the token */
+	width: 35ch !important;
 }
 
 .hidden {
```

Comment:
You can use the `hidden` attribute instead

Reply:
Was doing that, then realized that it has to be a separate PR (https://github.com/sindresorhus/notifier-for-github/pull/214#issuecomment-634475428). Requires a bit of changes in respective JS files.